.gitignore

```diff
@@ -1 +1,34 @@
-/*.local.php
+# phpstorm project files
+.idea
+
+# netbeans project files
+nbproject
+
+# zend studio for eclipse project files
+.buildpath
+.project
+.settings
+
+# windows thumbnail cache
+Thumbs.db
+
+# composer vendor dir
+/vendor
+
+/composer.lock
+
+# composer itself is not needed
+composer.phar
+
+# Mac DS_Store Files
+.DS_Store
+
+# phpunit itself is not needed
+phpunit.phar
```

Comment:
There is no phpunit.

Reply:
It may be needed in the future if we implement more commands and tools. I just added the most commonly used set of ignored files and folders. I propose to leave it as it is.

Reply:
We'll add it when needed.

Reply:
> We'll add it when needed.

OK, you're right.
Fixed :)